front/lib/api/agent_data_sources.ts

```diff
@@ -83,6 +83,9 @@ export async function getDataSourceViewsUsageByCategory({
     AgentDataSourceConfiguration.findAll({
       raw: true,
       group: ["dataSourceView.id"],
+      where: {
+        workspaceId: owner.id,
+      },
```

Comment:
this does not change the result of the query right ? already filtered by constraint on inner join on agent_configuration ?

Reply:
Yes, from the tests it was the same